jenkins_github_checkwarn.sh

```diff
@@ -88,7 +88,7 @@ if [ -n "${GERRIT_PROJECT-}" ]; then
   set -u
 fi
 
-if [ -x ./ci/patch_src_in_place ]
+if [ -f ./ci/patch_src_in_place ]
```

Comment:
Why does the PR still use `-x` (https://build.hpdd.intel.com/blue/rest/organizations/jenkins/pipelines/daos-stack/pipelines/code_review/branches/PR-29/runs/4/nodes/21/log/?start=0)?
```
[2022-01-19T14:10:27.767Z] + '[' -n '' ']'
[2022-01-19T14:10:27.767Z] + '[' -x ./ci/patch_src_in_place ']'
[2022-01-19T14:10:27.767Z] + python3 code_review/github_checkpatch.py
```

Reply:
You need to look at a later build

```
[2022-01-19T14:24:10.202Z] + '[' -f ./ci/patch_src_in_place ']'
[2022-01-19T14:24:10.202Z] + ./ci/patch_src_in_place
[2022-01-19T14:24:10.202Z] ++ git diff --name-only origin/master...
```

Reply:
Ah no, I see the issue.  The build for this PR?

Jenkinsfile calls checkPatch which is in the pipeline-lib repo, that checks out the code_review repo and calls it so the code used to check this PR is not the code that's in this PR.  https://github.com/daos-stack/daos/pull/7928 has the change in the jenkinsfile required to tell pipeline-lib which branch of code_review to use, I could presumably add that change to this PR and it would check itself, but I'd then need to revert that change before this could be landed.